supervision/annotators/core.py

```diff
@@ -1369,7 +1382,7 @@ def annotate(
                 xy=(text_x, text_y),
                 text=text,
                 font=self.font,
-                fill=self.text_color.as_rgb(),
+                fill=text_color.as_rgb(),
```

Comment:
In this, do we need to change `as_rgb()` to `as_bgr()` same like we did in LabelAnnotator?

Reply:
`RichLabelAnnotator` uses Pillow as a rendering engine. Pillow accepts colors in RGB order, while OpenCV in BGR. So we keep `as_rgb()` here.